site/content/docs/5.3/utilities/borders.md

```diff
@@ -149,6 +164,10 @@ Add classes to an element to easily round its corners.
 
 Use the scaling classes for larger or smaller rounded corners. Sizes range from `0` to `5`, and can be configured by modifying the utilities API.
 
+{{< design-callout-alert >}}
+These border variants with **a size css class different than `rounded-0`** should not be used because they do not respect the Orange Design System specifications.
```

Comment:
```suggestion
These border variants **other than `.rounded-0`** should not be used because they do not respect the Orange Design System specifications.
```

Reply:
OK kept the original sentence, modified "css" to "CSS", and added the dot in front of the classes. See https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2064/commits/73ea34c925f4cb946c15a6b64838d227cf739040.